lib/reports/cost_report.rb

```diff
@@ -81,11 +85,21 @@ def num_pd_volumes
       @num_pd_volumes ||= Cluster.collection.aggregate(
         [
           {"$match": {"ht_items.0": {"$exists": 1}}},
-          {"$group": {_id: nil,
-                      items_count: {"$sum": {"$size": {
-                        "$filter": {input: "$ht_items", as:    "item",
-                                    cond: {"$eq": ["$$item.access", "allow"]}}
-                      }}}}}
+          {"$group": {
+            _id: nil,
+            items_count: {"$sum": {"$size": {
+              "$filter": {
+                input: "$ht_items",
+                as: "item",
+                cond: {
+                  "$or": [
+                    {"$eq": ["$$item.access", "allow"]},
+                    {"$eq": ["$$item.rights", "icus"]}
```

Comment:
is there a test that fails without this change?

Reply:
No, good point.

Reply:
Wait, I'm still not addressing the thing in your feedback, am I.

Reply:
Broke out `ic_volumes` so I can actually count them, in order to address this bit of feedback.